adapters/http4s/src/main/scala/caliban/Http4sAdapter.scala

```diff
@@ -327,7 +324,7 @@ object Http4sAdapter {
       } yield response
     }
 
-  def makeWebSocketService[R, E](
+  def makeWebSocketService[R <: Has[_] with Clock with Blocking, E](
```

Comment:
Unfortunately, the compilation fails without ` <: Has[_] with Clock with Blocking` in Scala 3.

But compiles fine in 2.13.6.

Reply:
Where does the constraint come from?

Reply:
I tried your branch and was able to make it work: change the queues to be of `Task` instead of `RIO[R, *]` and it will solve the problem. You also need to put `.provideLayer(Clock.live)` back.

There's still an error but I think it's something else.

Reply:
I think I made it work, will open a PR to your repo.

Reply:
I was able to push to your branch directly, you can have a look